src/webui/api/torrentscontroller.cpp

```diff
@@ -55,8 +55,12 @@
 // Tracker keys
 const char KEY_TRACKER_URL[] = "url";
 const char KEY_TRACKER_STATUS[] = "status";
+const char KEY_TRACKER_TIER[] = "tier";
 const char KEY_TRACKER_MSG[] = "msg";
-const char KEY_TRACKER_PEERS[] = "num_peers";
+const char KEY_TRACKER_RECEIVED[] = "num_peers";
```

Comment:
TRACKER_RECEIVED == "num_peers" ?

Reply:
The value associated with this key is used to display the tracker "Received" count, but for legacy reasons the key is `num_peers`. We can't change this without breaking the api.

Reply:
Maybe you'd better add four new keys and leave "num_peers" for a deprecation later on?